modules/gdscript/gdscript_editor.cpp

```diff
@@ -434,6 +434,63 @@ void GDScriptLanguage::get_recognized_extensions(List<String> *p_extensions) con
 	p_extensions->push_back("gd");
 }
 
+void GDScriptLanguage::get_public_keywords(List<MethodInfo> *p_keywords) const {
+	Vector<String> keywords = {
+		// Control flow.
```

Comment:
It would be nice to add a comment with a link to `GDScriptLanguage::get_reserved_words()` and a note that function- and constant-like keywords are excluded.

https://github.com/godotengine/godot/blob/80db886ec251702aacec99d03628f6a0b8c44297/modules/gdscript/gdscript.cpp#L2719-L2721